views/graph/show.handlebars

```diff
@@ -1,110 +1,407 @@
+                })
+                .on('VRManInstructionsHidden', function () {
+                    fadingIn = fadingOut = false
+                    if (siteUI.isPortraitLike()) {
+                        headerFadein()
+                        queueFadeoutTimer()
+                    }
+                })
+
+            setTimeout(function () {
+                if (VizorUI.isMobile.any() && !siteUI.isPortraitLike())
+                    headerFadeout()
+                else
+                    queueFadeoutTimer()
+            }, 300)
+        }
+
+        function bindVRBackButton() {
+            $('#backfromvr').off('.vizor').on('click.vizor', function (e) {
+                E2.core.webVRManager.onBackClick_()
+                siteUI.tagBodyClass()
+                return false
+            })
+        }
+
+        function bindShareButton() {
+            var $share = $('button#sharebutton')
+            if ($share.length < 1) return
+
+            $share
+                .on('click', function (e) {
+                    e.preventDefault()
+                    var html = E2.views.partials.embedVRplayer(playerUI.data)
+                    suspendVRcamera()
+                    var modal = VizorUI.modalOpen(html, "Share this", 'player_share doselect_all', enableVRcamera)
+                    modal
+                        .find('textarea, input')
+                        .on('mouseup touchup', function (e) {
+                            e.currentTarget.select()
+                            e.currentTarget.setSelectionRange(0, 9999)
+                            e.preventDefault()
+                            return true
+                        })
+                        .on('focus', function (e) {
+                            e.preventDefault();
+                            e.stopPropagation();
+                            return false
+                        })
+
+                    return false;
+                })
+        }
+
+        function amendVRManagerInstructions() {
+            if (!(E2 && E2.core && E2.core.webVRManager && E2.core.webVRManager.rotateInstructions)) {
+                console.error('amend instructions called too early?', E2)
+                return
+            }
+
+            var r = E2.core.webVRManager.rotateInstructions
+            r.text.innerHTML += "<br /><br /><button id='backfromvr'>Exit VR view</button>"
+            r.overlay.style.background = '#2b2f37'
+            r.overlay.style.zIndex = "100"
+            $('div:last-child', r.overlay)[0].style.display = 'none'
+        }
+
+        function completeLoading() {
+            if (E2.util.isMobile()) {
+                // mobile
+            }
+            else if (/iPad/i.test(navigator.userAgent)) {
+                // tablet
+            }
+            else {
+                // desktop / default
+            }
+            $body.addClass('playing')
+            playerUI.selectStage('stage')
+            mixpanel.track('Player (autoplay)')
```

Comment:
Please use a better event name for tracking. What is happening here? "Player autoplaying"?